main.py

```diff
@@ -59,6 +61,26 @@ def get_city():
 
     return city.capitalize()
 
+def store_data():
+    db_handler = DBHandler('records_db.sqlite')
+
+    db_handler.create_table()# creates table if it does not exist
+
+#the following lines add data using dbhandler methods
+
+db_handler.add_travel_advisory(advisory_data)
+db_handler.addd_food_to_eat(food_data)
```

Comment:
I noticed this was mispelled.  I'll make note of it to fix this later as needed.